common/src/main/java/com/microsoft/identity/common/internal/commands/parameters/InteractiveTokenCommandParameters.java

```diff
@@ -74,6 +74,9 @@
     @Expose()
     private final boolean handleNullTaskAffinity;
 
+    @Expose()
+    private final boolean authorizationInCurrentTask;
```

Comment:
Would you mind adding comments to describe what this is for? (mainly for our future selves)